src/blockdev.rs

```diff
@@ -446,21 +439,14 @@ fn read_sysfs_dev_block_value(maj: u64, min: u64, field: &str) -> Result<String>
 
 pub fn lsblk(dev: &Path) -> Result<Vec<HashMap<String, String>>> {
     // Older lsblk, e.g. in CentOS 7.6, doesn't support PATH, but -p option
-    let result = Command::new("lsblk")
-        .arg("--pairs")
+    let mut cmd = Command::new("lsblk");
+    cmd.arg("--pairs")
         .arg("--paths")
         .arg("--output")
         .arg("NAME,LABEL,FSTYPE,TYPE,MOUNTPOINT")
-        .arg(dev)
-        .output()
-        .chain_err(|| "running lsblk")?;
-    if !result.status.success() {
-        // copy out its stderr
-        eprint!("{}", String::from_utf8_lossy(&*result.stderr));
-        bail!("lsblk of {} failed", dev.display());
-    }
-    let output = String::from_utf8(result.stdout).chain_err(|| "decoding lsblk output")?;
+        .arg(dev);
 
+    let output = cmd_output(&mut cmd).chain_err(|| "running lsblk")?;
```

Comment:
Not `runcmd_output!`?

Reply:
Prep for #304 where one of the arguments will be conditional.